support/src/main/java/org/jenkinsci/plugins/workflow/support/steps/ExecutorStepExecution.java

```diff
@@ -519,6 +520,49 @@ public String getUrl() {
             private static final long serialVersionUID = 1L;
         }
     }
+    
+    /**
+     * Looks for the executor, which is acquired by {@link ExecutorStep}.
+     * @param flowExecution Flow execution
+     * @return Found {@link Executor}. Null if it cannot be found
```

Comment:
:bug: does not make sense. What is this supposed to return when there are two `node` blocks active in `parallel`?

I think you need to make it take `StepExecution`, or just be an instance method on `ExecutorStepExecution`.